src/main/java/uk/gov/hmcts/divorce/divorcecase/model/access/SolicitorAndSystemUpdateAccess.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.divorce.divorcecase.model.access;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.SetMultimap;
+import uk.gov.hmcts.ccd.sdk.api.HasAccessControl;
+import uk.gov.hmcts.ccd.sdk.api.HasRole;
+import uk.gov.hmcts.ccd.sdk.api.Permission;
+
+import static uk.gov.hmcts.ccd.sdk.api.Permission.CRU;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SOLICITOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SYSTEMUPDATE;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+
+public class SolicitorAndSystemUpdateAccess  implements HasAccessControl {
```

Comment:
need a test for this class similar to what we have for other roles

Reply:
Added in https://github.com/hmcts/nfdiv-case-api/pull/1971/commits/c6c2f7d9f0e8298f49b7f49c2f5f6322561c8bd9